src/pages/homepage/homepage.jsx

```diff
@@ -0,0 +1,29 @@
+import React from "react";
+import { FeaturedProducts } from "../../components/featuredProducts/featuredProducts";
+import { Footer } from "../../components/footer/footer";
+import { HeroSection } from "../../components/heroSection/heroSection";
+import { Navbar } from "../../components/navbar/navbar";
+import { ProductByCategory } from "../../components/productByCategory/productByCategory";
+import "../../components/styles/main.css";
```

Comment:
create index file inside components folder and export all from there so you don't require to write this every time.